examples/consumer_example/consumer_example.go

```diff
@@ -81,11 +85,19 @@ func main() {
 
 			switch e := ev.(type) {
 			case *kafka.Message:
+				// Process the message received.
 				fmt.Printf("%% Message on %s:\n%s\n",
 					e.TopicPartition, string(e.Value))
 				if e.Headers != nil {
 					fmt.Printf("%% Headers: %v\n", e.Headers)
 				}
+
+				// We can store the offsets of the messages manually or let
+				// the library do it automatically based on the setting
+				// enable.auto.offset.store. Once an offset is stored, the
+				// library takes care of periodically committing it to broker.
```

Comment:
Can we make this as 
```suggestion
				// library takes care of periodically committing it to broker
				// when enable.auto.commit isn't set to false, default is true.
```

Reply:
Done, but changed language just a little